llvm/include/llvm/CodeGen/MachineModuleInfoImpls.h

```diff
@@ -36,6 +36,11 @@ class MachineModuleInfoMachO : public MachineModuleInfoImpl {
   /// bit is true if this GV is external.
   DenseMap<MCSymbol *, StubValueTy> ThreadLocalGVStubs;
 
+  /// Darwin '$auth_ptr' stubs.  The key is the stub symbol, like
+  /// "Lfoo$addend$auth_ptr$ib$12".  The value is the MCExpr representing that
```

Comment:
Just in case you've missed that - currently, there is no addend support for `$auth_ptr$` symbols - see `getAuthPtrSlotSymbolHelper`. It's probably worth mentioning that addend is optional in this comment. Feel free to ignore though.

Reply:
Right, I don't think we'll ever need the addend support here, I'm keeping it downstream for some other use-cases.

I should mention we're also missing the address-diversity support here, and that one we'll likely want to add here in the future, at least for MachO;  we'll see then whether it's needed for ELF, given the broader support for auth GOT relocations that MachO doesn't expose.